src/containers/Token/index.jsx

```diff
@@ -53,57 +59,32 @@ class Token extends Component {
     )
   }
 
-  constructor(props) {
-    super(props)
-    this.state = {}
-  }
-
-  componentDidMount() {
-    analytics(ANALYTIC_TYPES.pageview, { title: 'Accounts', path: '/accounts' })
-  }
-
-  componentWillUnmount() {
-    window.scrollTo(0, 0)
-  }
-
-  render() {
-    const { t, error, match } = this.props
-    const { prevId } = this.state
-    const accountId = match.params.id || ''
-    const currency = match.params.currency || ''
-    const showError = accountId === prevId && error
-
-    // TODO: title this so it includes the currency code somehow
-    document.title = `${t('xrpl_explorer')} | ${accountId.substr(0, 12)}...`
-
-    return showError ? (
-      Token.renderError(error)
-    ) : (
-      <div className="token-page">
-        {accountId && (
-          <TokenHeader accountId={accountId} currency={currency} t={t} />
-        )}
-        {accountId && IS_MAINNET && (
-          <DEXPairs accountId={accountId} currency={currency} t={t} />
-        )}
-        {accountId && (
-          <div className="section">
-            <h2>{t('token_transactions')}</h2>
-            <TokenTransactionTable accountId={accountId} currency={currency} />
-          </div>
-        )}
-        {!accountId && (
-          <div style={{ textAlign: 'center', fontSize: '14px' }}>
-            <h2>Enter an account ID in the search box</h2>
-          </div>
-        )}
-      </div>
-    )
-  }
+  return showError ? (
+    renderError(error)
```

Comment:
Why not refactor to a component? <Error

if(error)  {
  return <Error
}

return <div

Reply:
I will create a ticket to do this across the board.  These were existing methods and it is how it is done in many places throughout the app.

Reply:
Created #651